### PR TITLE
Backend changes many to many and one to many made by andrew

### PR DESCRIPTION
**Pull request by tatacsd**

### CODE REVIEW

**Comment by Andrew-shanan** [comment]
Ok it is fine

**Review by Igor-Koshevenko — APPROVED** [review]
It is running good. You have my russian approval
image

**Comment by Igor-Koshevenko** [comment]
@Andrew-shanan should the invoice api return this? 
image

File: PCU/src/main/java/com/coffeelovers/pcu/model/Invoice.java [hunk 1]
```diff
@@ -11,7 +15,7 @@
 	
 	@Id
 	@GeneratedValue(strategy = GenerationType.AUTO)
-	private long id;
+	private long invliceId;
```
Comment (luanrcard) [inline PCU/src/main/java/com/coffeelovers/pcu/model/Invoice.java:18]: @Andrew-shanan should this one be InvoiceId?

**Review by luanrcard — APPROVED** [review]
It's running. just found the potential typo in the Invoice class (invoiLceid).

**Comment by tatacsd** [comment]
I will merge since everything is working. and @Andrew-shanan can add his changes to it.

File: PCU/src/main/java/com/coffeelovers/pcu/model/Invoice.java (hunk 1 above)
Comment (tatacsd) [reply PCU/src/main/java/com/coffeelovers/pcu/model/Invoice.java:18]: @Andrew-shanan when you add the things you said you want to add you can fix it.

**Comment by Andrew-shanan** [comment]
> @Andrew-shanan should the invoice api return this? image

add api/ to the link to get the right json

**Comment by Andrew-shanan** [comment]
> @Andrew-shanan should the invoice api return this? image

ok i will correct it.

**Comment by Andrew-shanan** [comment]
> I will merge since everything is working. and @Andrew-shanan can add his changes to it.

thank you our leader, i will push some updates once everything is ok

**Pull request merged**